src/app/core/_components/forms/custom-forms/task/new-supertasks/new-supertasks.component.html

```diff
@@ -0,0 +1,25 @@
+<grid-main>
```

Comment:
Same as the previous templates, a mix of `[attribute]="''"` and `attribute=""` (I would prefer the last)

Reply:
Done